CHANGELOG.md

```diff
@@ -1,14 +1,33 @@
 # Snyk Vulnerability Scanner Changelog
 
 ## [Unreleased]
+
 ### Fixed
+
+### Changed
+- new tool window icon
+- new experimental welcome workflow for a limited audience
+
+## [2.4.5]
+
+### Fixed
+
 - run CLI download retries in background instead of UI thread
 - validate CLI download with sha-256
+- allow Snyk Code scan for multi-modules project (cause IlligalStateException before)
+
+### Changed
+
+- feedback link and message update
 
 ## [2.4.4]
+
 ### Fixed
+
 - Handle errors when downloading Snyk CLI
+
 ### Changed
+
```

Comment:
extra lines?

Reply:
Seems like what IntelliJ does automatically - I use the .editorconfig which governs our style